planet/Tests.ocf/Clouds.ocs/Map.c

```diff
@@ -0,0 +1,189 @@
+/**
+	Gem Grabbers
+	Dynamic map with sky islands and hard to reach gems. There is a main island, with at the bottom gem stalactites
+	and some smaller islands which also have gem stalactites hanging from the bottom. Each of the smaller islands
+	have different materials.
+	
+	@author Maikel
+*/
+
+#include Library_Map
+
+
+// Called be the engine: draw the complete map here.
+protected func InitializeMap(proplist map)
+{
+	// Retrieve the settings according to the MapSize setting.
+	var map_size, main_size, nr_islands;
+	/* if (SCENPAR_MapSize == 1) */
+	/* { */
+		map_size = [240, 200]; main_size = 80; nr_islands = RandomX(6, 7);
+	/* } */
+	/* if (SCENPAR_MapSize == 2) */
+	/* { */
+	/* 	map_size = [280, 220]; main_size = 90; nr_islands = RandomX(7, 8); */
+	/* } */
+	/* if (SCENPAR_MapSize == 3) */
+	/* { */
+	/* 	map_size = [320, 240]; main_size = 100; nr_islands = RandomX(8, 9); */
+	/* } */
```

Comment:
This code should either be made active, or removed entirely. (Also the comment style is strange.)